index.js

```diff
@@ -573,8 +574,23 @@ class Raspi extends EventEmitter {
     throw new Error('reportDigitalPin is not yet implemented');
   }
 
-  pingRead() {
-    throw new Error('pingRead is not yet implemented');
+  pingRead(config, handler) {
+    const pulseOut = config.pulseOut;
```

Comment:
I wasn't too sure myself - especially about pulseOut. I'm assuming it is used to indicate how often a reading should be taken from the ultrasonic sensor. But not sure what the unit is either. 5μs / 5ms seems too short and 5s seems too long.